cmd/snap-bootstrap/partition/mkfs_test.go

```diff
@@ -117,9 +117,14 @@ func (s *mkfsSuite) TestMakefilesystems(c *C) {
 			},
 		},
 	}
-	err := partition.MakeFilesystems(created)
-	c.Assert(err, IsNil)
-	c.Assert(s.mockMkfsExt4.Calls(), HasLen, 2)
+
+	for i, part := range created {
+		err := partition.MakeFilesystem(part)
+		c.Assert(err, IsNil)
+		c.Assert(s.mockMkfsVfat.Calls(), HasLen, 1)
+		c.Assert(s.mockMkfsExt4.Calls(), HasLen, i)
```

Comment:
Maybe a small comment here, when I skimmed over this (admittedly in the early morning) I missed that the first line has a "1" and the second a "i". Maybe something like `// single fat partition is created first, then 2 ext4 partitions`. But not a blocker :)

Reply:
Thanks. I think the difference is much more noticeable in the terminal font. Adding the comment, and also changing the variable to `n` to prevent misreadings.